server/blob/blob-storage-strategy/src/main/scala/org/apache/james/server/blob/deduplication/PassThroughBlobStore.scala

```diff
@@ -51,6 +52,15 @@ class PassThroughBlobStore @Inject()(blobStoreDAO: BlobStoreDAO,
       .`then`(SMono.just(blobId))
   }
 
+  override def save(bucketName: BucketName, data: ByteSource, storagePolicy: BlobStore.StoragePolicy): Publisher[BlobId] = {
```

Comment:
usually, starting at the third copy/paste, I factorize the code.

Reply:
I'd say let's drop the InputSream method.

Reply:
Fixups welcomed ;-)

The code is simple, easy to understand, tested, vary subtly in inputs and cannot easily find a way to factorize it hence will not spend my time on it.

Reply:
> Fixups welcomed ;-)

Come on, if you don't want the code reviewed, just say it.

Reply:
Come on, I ask kindly for suggestions, and you give none...